ansible_navigator/ui_framework/ui.py

```diff
@@ -477,16 +475,21 @@ def _serialize_color(self, obj: Any) -> CursesLines:
         :rtype: CursesLines
         """
         if self.xform() == "source.ansi":
-            return self._colorizer.render(doc=obj, scope=self.xform())
+            if self._ui_config.color:
+                return self._colorizer.render(doc=obj, scope=self.xform())
+            string = obj
         if self.xform() == "source.yaml":
             string = human_dump(obj)
         elif self.xform() == "source.json":
             string = json.dumps(obj, indent=4, sort_keys=True)
         else:
             string = obj
-        colorized = self._colorizer.render(doc=string, scope=self.xform())
-        lines = self._color_lines_for_term(colorized)
-        return lines
+        if self._ui_config:
+            rendered = self._colorizer.render(doc=string, scope=self.xform())
+        else:
+            # TODO: dedupe this from colorize.py
+            rendered = [[{"column": 0, "chars": doc_line, "color": None}] for doc_line in string.splitlines()]
```

Comment:
we could create a new scope called "no color" or something and move this over there